docs/dev/ci/github_actions/runners.md

```diff
@@ -1,43 +1,47 @@
-# Overview of the Runners used in the OpenVINO GitHub Actions CI
+# OpenVINO Runners used by GitHub Actions CI
 
-The machines that execute the commands from the workflows are referred to as _runners_ in GitHub Actions.
+The machines that execute workflow commands are referred to as _runners_ in GitHub Actions.
 
 Two types of runners are available in this repository:
-   
+
 * [GitHub Actions Runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners) - runners provided and managed by GitHub
-* [Self-hosted Runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) - runners created and managed by the OpenVINO CI team and linked to the OpenVINO repositories 
+* [Self-hosted Runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) - runners created and managed by the OpenVINO CI team and linked to the OpenVINO repositories
+
+The runners are specified for each job using the `runs-on` key.
 
-The runners are specified for each job using the `runs-on` key. 
+An example `Build` job from the [`linux.yml`](./../../../../.github/workflows/linux.yml)
+workflow, using the `aks-linux-16-cores-32gb` runner group:
 
-An example `Build` job from the [`linux.yml`](./../../../../.github/workflows/linux.yml) workflow:
 ```yaml
 Build:
   ...
   runs-on: aks-linux-16-cores-32gb
   ...
 ```
 
-The `aks-linux-16-cores-32gb` runners group is used for this job.
 
 ## Available GitHub Actions Runners
 
-GitHub provides runners with different combinations of available resources and software. 
+GitHub provides runners with different combinations of available resources and software.
+OpenVINO repositories make use of the following runners:
+
+* [default runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources),
+  used for not-so-intensive memory and CPU tasks: `ubuntu-22/20.04`, `windows-2019/2022`,
+  `macos-12/13`, etc.
 
-The OpenVINO repositories make use of the following runners:
+* [larger runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#machine-sizes-for-larger-runners),
+  used for memory and CPU heavy tasks, listed in [the runners page](https://github.com/openvinotoolkit/openvino/actions/runners).
```

Comment:
CPU-heavy or maybe CPU-intensive?
Like line 68: 
Jobs that are more memory and/or CPU-intensive require a more robust configuration.